tests/test_app.py

```diff
@@ -3,62 +3,20 @@
 import pytest
```

Comment:
FYI - We're keeping the `tests` because in the near future we will add the integration tests here.